src/bitmessagecurses/__init__.py

```diff
@@ -982,7 +983,9 @@ def sendMessage(sender="", recv="", broadcast=None, subject="", body="", reply=F
                         0,      # retryNumber
                         "sent",
                         2,      # encodingType
-                        BMConfigParser().getint('bitmessagesettings', 'ttl'))
+                        BMConfigParser().getint('bitmessagesettings', 'ttl')
```

Comment:
I would move this variable into `helper_sent` as well. Since you're already working on the code, may as well do it.

Reply:
ok